### PR TITLE
Add script to run OC tests

### PR DESCRIPTION
**Pull request by gtroitsk**
### Summary

(Summarize the problem solved by this PR, and how to verify it manually)

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Methods and classes used in PR scenarios are meaningful
- [ ] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: openshift-scripts/run-oc.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+mvn -B --no-transfer-progress -V -fae clean verify -Dlog.nocolor=true\
+        -P ${PROFILE} \
+        ${MVN_ADDITIONAL_ARGS} \
+        ${MVN_NATIVE_ARGS} \
+        ${MVN_QUARKUS_VERSION} \
+        -Dts.openshift.ephemeral.namespaces.enabled=true \
+        -Dopenshift -Dinclude.operator-scenarios \
+        -Dinclude.serverless \
+        -Dts.container.registry-url=quay.io/quarkusqeteam \
```
Comment (michalvavrik) [inline openshift-scripts/run-oc.sh:79]: I'd make this constant in Jenkins groovy constants class and pass it down as well, I'll comment in Jenkins gitlab PR soon.

File: openshift-scripts/run-oc.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+fi
+
+oc login ${OCP_URL} --username=${OCP_USERNAME} --password=${OCP_PASSWORD} --insecure-skip-tls-verify
+
+mvn -B --no-transfer-progress -V -fae clean verify -Dlog.nocolor=true\
+        -P ${PROFILE} \
+        ${MVN_ADDITIONAL_ARGS} \
+        ${MVN_NATIVE_ARGS} \
+        ${MVN_QUARKUS_VERSION} \
+        -Dts.openshift.ephemeral.namespaces.enabled=true \
```
Comment (michalvavrik) [inline openshift-scripts/run-oc.sh:76]: why do you set it? isn't it default?

**Comment by fedinskiy** [comment]
@gtroitsk are you planning to finish it this year (or at all)?

**Comment by gtroitsk** [comment]
> @gtroitsk are you planning to finish it this year (or at all)?

I am migrating script to Jenkins, I forgot to close this PR

**Pull request closed without merging**